frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js

```diff
@@ -55,128 +57,139 @@ function EditNetImportModalContent(props) {
       <ModalBody>
         {
           isFetching &&
-            <LoadingIndicator />
+          <LoadingIndicator />
```

Comment:
Standard form in the code base is to indent after conditional, this should be left alone

Reply:
I tried to make these changes, but got a ton of lint errors. so I just loaded it up in VS Code and ddid SHIFT-ALT F and it reverted the changes.

What is the best way I can configure vs code to lint automatically? With my last PR I had issues but linting with vs code seemed to resolve them. In this case, it seems VS code is not doing what we want?

Reply:
I usually don't let VSCode format the whole file and try to follow precisely what's already written. Maybe there is a way to auto-format and have the linter be okay with it. I dunno.